### PR TITLE
Changes for build and reproducibility

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Heads-up @Nowosad I think this fixes #459. Planning to merge it tomorrow pending any feedback. :disappointed_relieved:

**Comment by Robinlovelace** [comment]
Heads-up @Nowosad it seems to pass on travis but I have difficulty building the book locally with the M-K build that this branch uses, hence the more reliable approach of #462

File: 01-introduction.Rmd [hunk 1]
```diff
@@ -20,7 +20,7 @@ A simplistic comparison between the different approaches is illustrated in Table
 <!-- Still, many GIS programs ... but most teaching materials focus on the GUI ... to supplement the main graphical user interface (GUI), but  -->
 
 ```{r gdsl, echo=FALSE, message=FALSE}
-d = readr::read_csv("extdata/gis-vs-gds-table.csv")
+d = readr::read_csv("https://github.com/Robinlovelace/geocompr/raw/master/extdata/gis-vs-gds-table.csv")
```
Comment (Nowosad) [inline 01-introduction.Rmd]: @Robinlovelace this is going to break the PDF version.

File: 01-introduction.Rmd [hunk 2]
```diff
@@ -263,7 +263,7 @@ An obvious feature of the **tidyverse** is the tendency for packages to work in
 There is no equivalent **geoverse**, but there are attempts at harmonization between packages hosted in the [r-spatial](https://github.com/r-spatial/discuss/issues/11) organization and a growing number of packages use **sf**\index{sf} (Table \@ref(tab:revdep)). 
 
 ```{r revdep, echo=FALSE, message=FALSE}
-top_dls = readr::read_csv("extdata/top_dls.csv")
+top_dls = readr::read_csv("https://github.com/Robinlovelace/geocompr/raw/master/extdata/top_dls.csv")
```
Comment (Nowosad) [inline 01-introduction.Rmd]: @Robinlovelace this is going to break the PDF version.

File: 01-introduction.Rmd (hunk 1 above)
Comment (Robinlovelace) [reply 01-introduction.Rmd]: Good point. Happy to revert this. Same with the second point.

**Comment by Robinlovelace** [comment]
Overall what's your thinking on getting Travis working again @Nowosad ? Modifying this branch or creating a new branch and putting in a new PR are the two options I think. If we can get the build working with interactive maps, but not interactive maps that use `addProviderTiles()` as per https://github.com/rstudio/bookdown/issues/15#issuecomment-573776659, that would be ideal IMO.

**Comment by Nowosad** [comment]
I will try to make a new branch. I am working on it now.

**Comment by Robinlovelace** [comment]
Great. Your approach looks more promising @Nowosad so I'll close this PR for now.

**Pull request closed without merging**